src/spead.py

```diff
@@ -212,7 +212,7 @@ def __init__(self, from_string=None, id=None, name='', description='', shape=[],
                 if isinstance(ndarray, numpy.ndarray) or (isinstance(ndarray, tuple) and len(ndarray) == 2):
                     self.dtype_str = self._dtype_pack(ndarray)
                     self.shape = ndarray.shape if isinstance(ndarray, numpy.ndarray) else ndarray[1]
-                    self.size = numpy.multiply.reduce(self.shape)
```

Comment:
numpy.multiply.reduce is what was there, but I've changed it to numpy.product now.